macaroonbakery/codec.py

```diff
@@ -7,8 +7,7 @@
 from nacl.encoding import Base64Encoder
 from nacl.public import Box, PublicKey
 
-from macaroonbakery import BAKERY_V1, BAKERY_V2, BAKERY_V3
-from macaroonbakery.third_party import legacy_namespace, ThirdPartyCaveatInfo
+import macaroonbakery
 from macaroonbakery import checkers
```

Comment:
This makes the checkers package look like any other type in the bakery.
Perhaps:

    import macaroonbakery.checkers as checkers

might be a bit clearer?